video_v4l2.c

```diff
@@ -1355,6 +1355,23 @@ void v4l2_mutex_destroy(void) {
 #endif // HAVE_V4L2
 }
 
+static int v4l2_fps_set(struct context *cnt, struct video_dev *curdev) {
+
+    src_v4l2_t *vid_source = (src_v4l2_t *) curdev->v4l2_private;
+    struct v4l2_streamparm* setfps;
+
+    setfps = (struct v4l2_streamparm *) calloc(1, sizeof(struct v4l2_streamparm));
+    memset(setfps, 0, sizeof(struct v4l2_streamparm));
```

Comment:
I know that I have introduced more than a few of these into the code but to keep your PR clean, you may want to think about eliminating this `memset`.  The statement immediately preceding this is a `calloc` which will set the struct 0's and therefore the `memset` is redundant.  The `mymalloc` is also available which does a `calloc` with some error checks.

Also, has this been checked with valgrind to ensure it is appropriately freed upon exit of the function?